src/utils/encryptor.cpp

```diff
@@ -160,14 +160,10 @@ Encryptor::decrypt(std::string_view password,
     try {
         aead->finish(decryptedData);
     } catch (const Botan::Integrity_Failure& e) {
```

Comment:
We probably could do away with the try/catch blocks altogether here since the catch block is only rethrowing the current exception. However, I sort of like how explicit it is. Let's me know what to check for when using your code.